src/components/shared/Button/Close.css

```diff
@@ -32,12 +32,24 @@
   background-color: var(--theme-selection-background);
 }
 
+.close-btn:focus {
+  background-color: var(--theme-selection-background);
```

Comment:
You also need the same styling like the one on line 27 but for when the button has :focus . Right now the 'x' has a gray background where it should be white.

Reply:
Styling for line 27, is hardcoding the background to white for the image, so we have a white x on a blue background. Are you saying, that the entire button area should be white 
![screenshot-2017-12-7 debugger](https://user-images.githubusercontent.com/6732578/33726513-3d485734-db43-11e7-8260-7ec994255f2d.png)
^ Like so when focused?